gms/gossiper.cc

```diff
@@ -1647,7 +1659,8 @@ bool gossiper::is_normal(const inet_address& endpoint) const {
 }
 
 bool gossiper::is_left(const inet_address& endpoint) const {
-    return get_gossip_status(endpoint) == sstring(versioned_value::STATUS_LEFT);
+    auto status = get_gossip_status(endpoint);
+    return status == sstring(versioned_value::STATUS_LEFT) || status == sstring(versioned_value::REMOVED_TOKEN);
```

Comment:
Please rename the function to `is_left_or_removed` if you want to do this change.

Reply:
The is_left is in the sense of node left the cluster, no matter it is decommissioned or removed (It is not important for the caller to know the detail). We have another check is_normal_ring_member which checks if the node is in normal or shutdown status, but we are not naming it as is_normal_or_shutdown.